caikit_nlp/modules/text_generation/text_generation.py

```diff
@@ -161,22 +168,21 @@ def save(self, model_path):
             model_path: str
                 Folder to save text-generation caikit model
         """
-        saver = ModuleSaver(
-            self,
-            model_path=model_path,
-        )
+        saver = ModuleSaver(self, model_path=model_path)
```

Comment:
Actually not related to this PR, but `AutoConfig` at line 132 will always try to connect to the internet so can cause issues.  This was fixed in prompt tuning training, but not here. Might be worth including in this PR since it can potentially break model running. 

Solution: https://github.com/caikit/caikit-nlp/blob/main/caikit_nlp/modules/text_generation/peft_prompt_tuning.py#L393

Reply:
Nice, good catch!